### PR TITLE
Portshift Operator

### PR DESCRIPTION
**Pull request by FrimIdan**

### CODE REVIEW

**Comment by PennyScissors** [comment]
@FrimIdan Can you take me through the steps for install this chart? I'm not sure where I can get the Portshift Operator's access key, secret key and cluster ID to successfully install it.

**Comment by FrimIdan** [comment]
> @FrimIdan Can you take me through the steps to install this chart? I'm not sure where I can get the Portshift Operator's access key, secret key and cluster ID to successfully install it.

@PennyScissors , all the relevant info can be retried from Portshift console.
Do you need us to open you a test account to try it out or just need to understand where you can get the info from?

**Comment by FrimIdan** [comment]
@PennyScissors did you had the time to review it? Any more info is needed?

File: charts/portshift-operator/0.1.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - portshift
+  - operator
+  - monitoring
+  - security
+  - alerting
+  - metric
+  - troubleshooting
+  - run-time
+home: https://www.portshift.io/
+icon: https://www.portshift.io/wp-content/uploads/2019/10/portshift-logo-68.png
```
Comment (PennyScissors) [inline charts/portshift-operator/0.1.0/Chart.yaml:17]: Ideally, this file should be local in the `portshift-operator` folder and referenced here as `file://../logo.png`. This way the logo can be used in airtight environments.

**Pull request merged**